common/src/main/java/tachyon/conf/TachyonConf.java

```diff
@@ -219,6 +219,8 @@ public void merge(TachyonConf alternateConf) {
 
   // TODO(binfan): this method should be hidden and only used during initialization and tests.
   public void set(String key, String value) {
+    Preconditions.checkNotNull(value != null,
```

Comment:
I thought about it. It's very unlikely we call set with null key, but we can add it anyway.